app/assets/stylesheets/course/assessment/submission/submissions.scss

```diff
@@ -11,4 +11,14 @@
       padding: 0 3px;
     }
   }
+
+  &.index {
+    .submissions {
+      .no-submission {
+        strong {
+          color: $red;
```

Comment:
uhmmmm.

I'm not sure if this is getting a *bit* too extreme. `color: red` would work...